### PR TITLE
Bookmarkpage

### PR DESCRIPTION
**Pull request by therealchrismcfly**
There is now a bookmark icon on every exercise card. When the bookmark icon is clicked, it is filled and the exercise card is added to the bookmark page. You can see the bookmarked exercises on the new bookmark page in the Navbar. You can remove an exercise card from the bookmark page when you click the bookmark icon again.

### CODE REVIEW

**Comment by RobertBReich** [comment]
😮

File: src/components/Card/favoritesCard.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+					)}
+				</BookmarkButton>
+				<StyledCalendarButton
+					onClick={() => {
+						setIsShown(!isShown);
+					}}
+				>
+					add to planner
+				</StyledCalendarButton>
+				{isShown ? <IndexCalendar id={card.id} /> : ''}
```
Comment (JessicaLoers) [inline src/components/Card/favoritesCard.js:74]:
```suggestion
				{isShown && <IndexCalendar id={card.id} /> }
```

File: src/components/Card/favoritesCard.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+						handleChange();
+					}}
+				>
+					{buttonText ? 'Show instruction' : 'Hide instruction'}
+				</StyledHideButton>
+				{showText ? (
+					<StyledCardDescription id={card.id}>{card.description}</StyledCardDescription>
+				) : (
+					''
+				)}
```
Comment (JessicaLoers) [inline src/components/Card/favoritesCard.js:90]:
```suggestion
				{showText && <StyledCardDescription id={card.id}>{card.description}</StyledCardDescription> }
```

File: src/components/Buttons/BookmarkButton/filled.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const FilledBookmarkIcon = () => {
+	return (
+		<svg style={{width: '24px', height: '24px'}} viewBox="0 0 24 24">
+			<path
+				fill="currentCredolor"
```
Comment (JessicaLoers) [inline src/components/Buttons/BookmarkButton/filled.js:5]:
```suggestion
				fill="currentColor"
```

File: src/components/Card/indexCard.js [hunk 4]
```diff
@@ -47,6 +53,18 @@ export default function IndexCard({card}) {
...
+						bookmarked(card.id);
+						setIsBookmarked(!isBookmarked);
+					}}
+				>
+					{cards[card.id].isBookmarked ? (
+						<FilledBookmarkIcon id={card.id} />
+					) : (
+						<NotFilledBookmarkIcon id={card.id} />
+					)}
+				</BookmarkButton>
```
Comment (JessicaLoers) [inline src/components/Card/indexCard.js:67]: As we mentioned you could think about using a single SVG component - but that's a task for "Future Chris" 😁

**Review by JessicaLoers — APPROVED** [review]
**Hi @therealchrismcfly**  🙂

I have left you a few notes we have already talked about.

**Pull request merged**